Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -41,21 +41,37 @@ public class BTShopperInsightsClient {
             let postParameters = BTEligiblePaymentsRequest(
                 email: request.email,
                 phone: request.phone,
-                merchantID: "TODO-merchant-id-type"
+                merchantID: "MXSJ4F5BADVNS"
             )
             
             do {
-                let (_, _) = try await apiClient.post("/v2/payments/find-eligible-methods", parameters: postParameters, httpType: .payPalAPI)
-                
-                // TODO: - Handle API Response. DTBTSDK-3388
-                let result = BTShopperInsightsResult()
+                let (json,_) = try await apiClient.post("/v2/payments/find-eligible-methods", parameters: postParameters, httpType: .payPalAPI)
+                let eligibleMethodsJSON: BTJSON = json?["eligible_methods"] ?? BTJSON()
+                let eligibilePaymentMethods = BTEligibilePaymentMethods(json: eligibleMethodsJSON)
+                let result = BTShopperInsightsResult(
+                    isPayPalRecommended: isPaymentRecommended(eligibilePaymentMethods.paypal),
+                    isVenmoRecommended: isPaymentRecommended(eligibilePaymentMethods.venmo)
+                )
                 return self.notifySuccess(with: result)
             } catch {
                 throw self.notifyFailure(with: error)
             }
         }
     }
     
+    /// This method determines whether a payment source is recommended
+    /// - Parameters:
+    ///    - paymentMethodDetail: a `BTEligiblePaymentMethodDetails` containing the payment source's information
+    /// - Returns: `true` if both `eligibleInPPNetwork` and `recommended` are enabled, otherwise returns false.
+    /// - Note: This feature is in beta. It's public API may change or be removed in future releases.        -
```

Comment:
👏 This is so awesome and so well written! Ideally all of our functions are documented as well as this

Nit - since it's private we can remove that "beta" note.
```suggestion
    /// This method determines whether a payment source is recommended
    /// - Parameters:
    ///    - paymentMethodDetail: a `BTEligiblePaymentMethodDetails` containing the payment source's information
    /// - Returns: `true` if both `eligibleInPPNetwork` and `recommended` are enabled, otherwise returns false.
```